CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Implemented a simple gradient descent post-processing step, as described in the paper.
+  - The new post-processing step is similar to Langevin dynamics but without noise.
+  - Implemented using the Euler method with box constraint imposition at each iteration.
+  - This enhancement is designed to reflect the results outlined in the paper.
```

Comment:
Done. I've also added a breaking changes section to the CHANGELOG. I will wait for @MarkoB4t1qbit 's response before resolving the conversation.

Reply:
Right. Thanks for adding that section! According to our new versioning standards, that means our next release should increment the major version.

Reply:
You can consider "box constraint" to be a two-word noun. On its own, we would never have hyphenated it in the paper. Even used as an adjective, like in "box constraint imposition", there is no need to add a hyphen, because nobody would read it as a "constraint imposition" of the type "box" (i.e., with "box" as a modifier).

Reply:
Okay, thanks! In that case, @Estella-Wang-1qbit feel free to resolve this :)